src/ellipticLCS.jl

```diff
@@ -116,8 +117,7 @@ struct LCSParameters
     function LCSParameters(
                 boxradius::Real,
                 indexradius::Real=1e-1boxradius,
-                combine_pairs::Bool=true,
-                combine_31::Bool=false,
+                merge_heuristics=[combine_20],
```

Comment:
The default above is to have both heuristics?

Reply:
No default is to have just combine_20

Reply:
okay, then in line 111 we need to remove `combine_31`?

Reply:
Ah, sorry, I messed it up. I overlooked that the above is from the ocean_example, where we can, of course, overwrite the defaults.